docs/getting-started/deployment.md

```diff
@@ -155,98 +162,6 @@ clusters using the kind tool as described in the
     java -jar cs.jar castor get-telemetry <#>
     ```
 
-### Upload Offline Material
```

Comment:
As tuples are no longer actively uploaded by the user, should there be a section / comment about when the VC is actually ready for use or how to check number of tuples (if still possible).
Is it expected to have enough tuples available within seconds or minutes (though it might hardly depend on system resources)?

Reply:
Add an admonition to make it clear that one has to wait until tuples are available.